boards/sparkfun/micromod/micromod_nrf52840.dts

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (c) 2024 Jerónimo Agulló <jeronimoagullo97@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "micromod_nrf52840-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "Sparkfun Micrmod nRF52840 board";
+	compatible = "sparkfun,micromod-nrf52840";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,ieee802154 = &ieee802154;
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		watchdog0 = &wdt0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 13 0>;
+			label = "Blue LED";
+		};
+	};
+
+	micromod_header: connector {
+		compatible = "sparkfun,micromod-gpio";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <0 0 &gpio0 4 0>,	/* A0 */
+			   <1 0 &gpio0 5 0>,	/* A1 */
+			   <2 0 &gpio0 27 0>,	/* D0 */
+			   <3 0 &gpio1 8 0>,	/* D1/CAM_TRIG */
+			   <4 0 &gpio0 15 0>,	/* I2C_INT# */
+			   <5 0 &gpio0 29 0>,	/* G0/BUS0 */
+			   <6 0 &gpio0 3 0>,	/* G1/BUS1 */
+			   <7 0 &gpio1 13 0>,	/* G2/BUS2 */
+			   <8 0 &gpio1 12 0>,	/* G3/BUS3 */
+			   <9 0 &gpio1 11 0>,	/* G4/BUS4 */
+			   <10 0 &gpio0 17 0>,	/* G5/BUS5 */
+			   <11 0 &gpio1 6 0>,	/* G6/BUS6 */
+			   <12 0 &gpio1 4 0>,	/* G7/BUS7 */
+			   <13 0 &gpio1 14 0>,	/* G8 */
+			   <14 0 &gpio0 9 0>,	/* G9/ADC_D-/CAM_HSYNC */
+			   <15 0 &gpio0 10 0>,	/* G10/ADC_D+/CAM_VSYNC */
+			   //<16 NO CONNECTED	/* G11/SWO */
+			   <17 0 &gpio0 20 0>;	/* SPI_CS */
+	};
+};
+
+&adc {
+	status = "okay";
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uicr {
+	nfct-pins-as-gpios;
+	gpio-as-nreset;
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uarte";
+	current-speed = <115200>;
+	status = "okay";
+	pinctrl-0 = <&uart0_default>;
+	pinctrl-1 = <&uart0_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+&uart1 {
+	compatible = "nordic,nrf-uart";
```

Comment:
This is incorrect. `uart1` cannot be "nordic,nrf-uart", only "nordic,nrf-uarte" (see [UART instances](https://docs.nordicsemi.com/bundle/ps_nrf52840/page/uart.html#d1445e350) and [UARTE instances](https://docs.nordicsemi.com/bundle/ps_nrf52840/page/uarte.html#d1474e397) in nRF52840).

Reply:
you are rigth, I will remove this line since it is defined as UARTE in the `nrf52840.dtsi`

Reply:
@anangl done!